hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/HollowAccountFinalizationSuite.java

```diff
@@ -557,6 +560,12 @@ final Stream<DynamicTest> hollowAccountCompletionWithEthereumContractCreate() {
                             getTxnRecord(TRANSFER_TXN_2).andAllChildRecords().logged();
 
                     allRunFor(spec, op2, op3, hapiGetSecondTxnRecord);
+                    // ensure that the finalized contract has a self management key
```

Comment:
Does a check of this kind need to be made above in `hollowAccountCreationWithContractCreate` - it's a HAPI contract create but an admin key is specified - or do just assume that works as it is tested elsewhere?

Reply:
`hollowAccountCreationWithContractCreate` does not finalize the hollow account through the same flow that results in a contract admin key

Reply:
ok, so since not hollow account finalization then not a proper subject for this Suite